README.md

```diff
@@ -2,77 +2,18 @@
 
 A pluggable WordPress login flow for Android.
 
-## Usage ##
-
-To use this library in your project, you must set it up as a subtree.
-From the root of your main project, add the subtree:
-
-    $ git subtree add --prefix=libs/login git@github.com:wordpress-mobile/WordPress-Login-Flow-Android.git develop
-
-This will create a new directory, `libs/login`, containing the contents of this repository.
-
-Next, you need to generate the `gradle.properties` file:
-
-    $ cp libs/login/gradle.properties-example libs/login/gradle.properties
-
-Configure the fields in `gradle.properties` to match the `compileSdkVersion`,
-`buildToolsVersion`, `targetSdkVersion`, and support library versions used
-by your project.
-
-At this point, you can add it to your project's root `settings.gradle`:
+## Usage
```

Comment:
I love how those changes simplify the `README` drastically, making it way simpler now 🍾